config/manager/deployment.yaml

```
@@ -67,3 +68,20 @@ spec:
             requests:
               cpu: 500m
               memory: 512Mi
+
+---
+apiVersion: v1
+kind: Service
+metadata:
+  annotations:
+    prometheus.io/port: "9998"
+    prometheus.io/scheme: https
```

whopsy, it's actually `http` by default 😅  

```suggestion
    prometheus.io/scheme: http
```